db/seeds.rb

```diff
@@ -0,0 +1,53 @@
+require 'Faker'
+
+mentor = User.create!(username: Faker::Internet.user_name,
+                      email: Faker::Internet.email,
+                      password_digest: Faker::Internet.password,
+                      bio: Faker::Lorem.sentences
+                      )
+
+troll = User.create!(username: Faker::Internet.user_name,
+                     email: Faker::Internet.email,
+                     password_digest: Faker::Internet.password,
+                     bio: Faker::Lorem.sentences
+                     )
+
+civic = User.create!(username: Faker::Internet.user_name,
+                     email: Faker::Internet.email,
+                     password_digest: Faker::Internet.password,
+                     bio: Faker::Lorem.sentences
+                     )
+
+
+10.times do
+  user = User.create!(username: Faker::Internet.user_name,
+                      email: Faker::Internet.email,
+                      password_digest: Faker::Internet.password,
+                      bio: Faker::Lorem.sentences
+                      )
+
+
+
+  question = user.questions.create!( title: Faker::Lorem.word,
```

Comment:
the chaining of methods to create a new resource here is AWESOME. you should always chain the creating of something new based on what it belongs to.